cylc/flow/config.py

```diff
@@ -670,10 +685,10 @@ def prelim_process_graph(self) -> None:
             all(item in ['graph', '1', 'R1'] for item in graphdict)
         ):
             # Pure acyclic graph, assume integer cycling mode with '1' cycle
-            self.cfg['scheduling']['cycling mode'] = INTEGER_CYCLING_TYPE
             for key in ('initial cycle point', 'final cycle point'):
                 if key not in self.cfg['scheduling']:
                     self.cfg['scheduling'][key] = '1'
+            self.cfg['scheduling']['cycling mode'] = INTEGER_CYCLING_TYPE
```

Comment:
Why this change?

Reply:
This is part of [Minor fix to ensure order of ICP vs cycling mode in processed config does not change on reload vs start](https://github.com/cylc/cylc-flow/pull/7048/commits/a398b6008941b75c42823e80b179d7661f50c5a9)

Due to `schd._set_workflow_params()` moving earlier on, `[scheduling]initial cycle point` will now be set in the config before the cycling mode on reload, so will appear first in flow-processed.cylc